logging/logger.go

```diff
@@ -42,15 +53,11 @@ func NewLogger(logLevel string) logr.Logger {
 				fmt.Fprintf(os.Stderr, "Failed to parse --log-level=%s: %v", logLevel, err)
 				os.Exit(1)
 			}
-
 			// For example, --log-level=debug a.k.a --log-level=-1 maps to zaplib.DebugLevel, which is associated to logr's V(1)
 			// --log-level=-2 maps the specific custom log level that is associated to logr's V(2).
 			level := zapcore.Level(levelInt)
 			atomicLevel := zaplib.NewAtomicLevelAt(level)
 			o.Level = &atomicLevel
 		}
-		o.TimeEncoder = zapcore.TimeEncoderOfLayout(time.RFC3339)
-	})
-
-	return log
+		return zap.New(zap.UseFlagOptions(o))
```

Comment:
I see extra indentations in this function. Would you mind trying to run `go fmt`?